core/src/main/kotlin/me/frmr/github/repopolicy/core/PolicyEngine.kt

```diff
@@ -96,7 +100,7 @@ class PolicyEngine(val policy: PolicyDescription, val logging: Boolean) {
       q += "org:${subject.org} "
     }
 
-    return searchRequest.q(q).list()
+    return searchRequest.q(q).list().toList()
```

Comment:
Any concerns about how this is going to scale? We're effectively forfeiting pagination and loading all the results in memory at once.

Reply:
Yeah, I thought about that. On the one hand, if we use pagination we can't have a long running (over 1 hour) job without adding a little more complexity to that method. On the other hand, there could be scale problems if we are dealing with 1000s of repos.